compiler/sigmatch.nim

```diff
@@ -2234,8 +2234,14 @@ proc paramTypesMatch*(m: var TCandidate, f, a: PType,
         # XXX this is still all wrong: (T, T) should be 2 generic matches
         # and  (int, int) 2 exact matches, etc. Essentially you cannot call
         # typeRel here and expect things to work!
-        let r = typeRel(z, f, arg[i].typ)
+        var r = typeRel(z, f, arg[i].typ)
         incMatches(z, r, 2)
+        if r == isNone:
+          if userConvMatch(c, m, f, a, arg) == nil and f.kind == tyVarargs:
+            if f.n == nil:
+              let baseRel = typeRel(m, base(f), a)
+              if baseRel == isGeneric:
```

Comment:
Should other cases be handled as well? When should it be a match but isn't `isGeneric`?

Reply:
Bump

Reply:
I don't know.